event-sourcing/event-source-api/src/main/java/uk/gov/justice/services/eventsourcing/source/api/service/EventStreamsFeedService.java

```diff
@@ -18,21 +19,24 @@
 @ApplicationScoped
 public class EventStreamsFeedService implements FeedService<EventStreamEntry> {
 
-    @Value(key = "streamsFeedSize", defaultValue = "25")
-    protected long pageSize;
-
     @Inject
     private PaginationCapableRepository<EventStream> repository;
 
     private FeedGenerator<EventStream, EventStreamEntry> feedGenerator;
 
+    private FeedMaxMinProvider<EventStreamEntry> feedMaxMinProvider;
```

Comment:
put a TODO on this for implementation.. or now we kept the other class we can reset this from null?

Reply:
added todo